content/applications/productivity/calendar.rst

```diff
@@ -9,3 +10,246 @@ Calendar
+  left empty, the meeting is considered to be taking place online.
+- :guilabel:`Manual Confirmation`: If :guilabel:`Resources` has been selected in the
+  :guilabel:`Availability On` field, tick the checkbox and enter a maximum percentage of the
+  selected resource(s)' total capacity to create a manual confirmation requirement to finalize the
+  meeting.
+- :guilabel:`Videoconference Link`: Select from :guilabel:`Odoo Discuss` or :guilabel:`Google Meet`
+  to include a video conference link in the meeting invitation.
+- :guilabel:`Up-front Payment`: Tick the checkbox to require users to pay before confirming their
+  booking. Once this is ticked, a link appears to :guilabel:`Configure Payment Providers`,
+  which enables online payments.
+- :guilabel:`Limit to Work Hours`: If :guilabel:`Users` has been selected in the
+  :guilabel:`Availability On` field, tick the checkbox to limit meeting time slots to the selected
+  :doc:`users' working hours <../hr/employees/new_employee>`.
+- :guilabel:`Create Opportunities`: When this is selected, each scheduled appointment creates
+  a new CRM opportunity.
+- :guilabel:`Reminders`: Add or delete notification reminders in this field. Select the blank space
+  for additional options.
+- :guilabel:`Confirmation Email`: Tick the checkbox to automatically send a confirmation email to
+  attendees once the meeting is confirmed. Select from the email templates or click
+  :guilabel:`Search More`, then :guilabel:`New` to create a custom template.
+- :guilabel:`Cancelation Email`: Tick the checkbox to automatically send a cancelation email to
+  attendees if the meeting is canceled. Select from the email templates or click
+  :guilabel:`Search More`, then :guilabel:`New` to create a custom template.
+- :guilabel:`CC to`: Add contacts to be notified of meeting updates in this field, regardless if
+  they attend the meeting.
+- :guilabel:`Allow Guests`: Tick the checkbox to allow invitees to invite guests.
+
+Questions
+~~~~~~~~~
+
+In the :guilabel:`Questions` tab, add questions for the invitee to answer when confirming their
+meeting. Questions can be configured for question type, placeholder answers, and whether they are
+required. To learn how to create more comprehensive questionnaires, head to the **Survey** app
+documentation on
+:doc:`creating and configuring data-capturing questions <../marketing/surveys/questions>`.
+
+Messages
+~~~~~~~~
+
+In the :guilabel:`Introduction Message` field of the :guilabel:`Messages` tab, add additional
+meeting information that appears on the invitation.
+
+Information added to the :guilabel:`Extra Message on Confirmation` field appears once the
+meeting is confirmed.
+
+Click the :guilabel:`Preview` button to see how the appointment link looks for attendees.
+
+Once the configurations are finished, click the :guilabel:`Share` button to generate a link to send
+directly, or click :guilabel:`Publish` to publish the appointment selection on the connected Odoo
+website.
```

Comment:
Suggested previously to move this before the "tab" sections, as there is no guarantee the reader configured the Messages
```suggestion
```